install/mysql/glpi-empty.sql

```diff
@@ -4361,6 +4361,7 @@ CREATE TABLE `glpi_networknames` (
   `name` varchar(255) DEFAULT NULL,
   `comment` text,
   `fqdns_id` int NOT NULL DEFAULT '0',
+  `ipnetworks_id` int(11) NOT NULL DEFAULT '0',
```

Comment:
You missed the key here (ok in migration)

Reply:
done